### PR TITLE
testsuite: wrap _{l,}stat64_time64 in test

### PR DESCRIPTION
**Pull request by zhsj**
Fix building with -D_FILE_OFFSET_BITS=64 -D_TIME_BITS=64 on 32bit arch.

Closes: #37
Bug: https://bugs.debian.org/1065973

### CODE REVIEW

**Comment by lucasdemarchi** [comment]
```
testsuite/path.c:206:45: warning: 'struct __stat64_t64' declared inside parameter list will not be visible outside of this definition or declaration
  206 | WRAP_2ARGS(int, -1, __stat64_time64, struct __stat64_t64*);
      |                                             ^~~~~~~~~~~~
testsuite/path.c:124:42: note: in definition of macro 'WRAP_2ARGS'
  124 | TS_EXPORT rettype name(const char *path, arg2t arg2)    \
      |                                          ^~~~~
testsuite/path.c:206:21: warning: no previous prototype for '__stat64_time64' [-Wmissing-prototypes]
  206 | WRAP_2ARGS(int, -1, __stat64_time64, struct __stat64_t64*);
      |                     ^~~~~~~~~~~~~~~
testsuite/path.c:124:19: note: in definition of macro 'WRAP_2ARGS'
  124 | TS_EXPORT rettype name(const char *path, arg2t arg2)    \
      |                   ^~~~
testsuite/path.c:207:46: warning: 'struct __stat64_t64' declared inside parameter list will not be visible outside of this definition or declaration
  207 | WRAP_2ARGS(int, -1, __lstat64_time64, struct __stat64_t64*);
      |                                              ^~~~~~~~~~~~
testsuite/path.c:124:42: note: in definition of macro 'WRAP_2ARGS'
  124 | TS_EXPORT rettype name(const char *path, arg2t arg2)    \
      |                                          ^~~~~
testsuite/path.c:207:21: warning: no previous prototype for '__lstat64_time64' [-Wmissing-prototypes]
  207 | WRAP_2ARGS(int, -1, __lstat64_time64, struct __stat64_t64*);
      |                     ^~~~~~~~~~~~~~~~
testsuite/path.c:124:19: note: in definition of macro 'WRAP_2ARGS'
  124 | TS_EXPORT rettype name(const char *path, arg2t arg2)    \
      |                   ^~~~

```

File: testsuite/path.c [hunk 1]
```diff
@@ -202,6 +203,8 @@ WRAP_2ARGS(int, -1, stat, struct stat*);
 WRAP_2ARGS(int, -1, lstat, struct stat*);
 WRAP_2ARGS(int, -1, stat64, struct stat64*);
 WRAP_2ARGS(int, -1, lstat64, struct stat64*);
+WRAP_2ARGS(int, -1, __stat64_time64, struct __stat64_t64*);
```
Comment (jspricke) [inline testsuite/path.c:206]:
```suggestion
struct __stat64_t64;
extern int __stat64_time64 (const char *file, struct __stat64_t64 *buf);
extern int __lstat64_time64 (const char *file, struct __stat64_t64 *buf);
WRAP_2ARGS(int, -1, __stat64_time64, struct __stat64_t64*);
```
(I did not find an easy way to get those prototypes from `sys/stat.h`. The `WRAP_2ARGS` is unmodified, I just add those three lines above.)

**Comment by jspricke** [comment]
@lucasdemarchi could you have an other look? Thanks!

**Comment by lucasdemarchi** [comment]
I did some cosmetic adjustments, added a commit on top to also fix a warning in a printf in depmod and pushed. Thanks.

**Pull request closed without merging**